src/services/device.service.ts

```diff
@@ -0,0 +1,23 @@
+import { db } from './database';
+
+export interface Device {
+  deviceId: number;
+  userId: number;
+  deviceToken: number;
+}
+
+interface InsertDeviceResult{
+    device_id: number;
+}
+
+const addDevice = async (device: Device): Promise<number> => {
+  const result = await db.sql<InsertDeviceResult>('insert-devices', {
+    user_id: device.userId,
+    device_token:device.deviceToken,
+  });
+  return result.rows[0].device_id;
```

Comment:
JSONifying this bare number will cause the same problem we're [experiencing with creating a notification](https://bitbucket.org/permanent-org/back-end/pull-requests/3#comment-215883439); that is, returning a bare number is not what we should be doing.

Instead, let's return the input back to the caller; as a RESTful API we should return something, and in this case the user ID and device token seem reasonable.

(I'm commenting here, but that will also affect the `InsertDeviceResult` interface and the SQL query.)